### PR TITLE
monitor: fix platform-level statistics

### PR DESCRIPTION
**Pull request by huanggze**
Signed-off-by: huanggze <loganhuang@yunify.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/models/monitoring/monitoring.go [hunk 1]
```diff
@@ -182,7 +182,13 @@ func (mo monitoringOperator) GetKubeSphereStats() Metrics {
 		})
 	}
 
-	tmpls, err := mo.op.ListApps(&params.Conditions{}, "", false, 0, 0)
+	cond := &params.Conditions{
+		Match: map[string]string{
+			"status":  "active",
```
Comment (zryfish) [inline pkg/models/monitoring/monitoring.go:187]: Try not to hardcode these variables, it's bad practice

Comment (huanggze) [reply pkg/models/monitoring/monitoring.go:187]: they are fixed values

Comment (huanggze) [reply pkg/models/monitoring/monitoring.go]: done

**Pull request merged**